frontend/src/lib/lemon-ui/Popover/Popover.tsx

```diff
@@ -51,18 +45,16 @@ export interface PopoverProps {
      * Works also with strings, matching classnames or ids, for antd legacy components that don't support refs
      * **/
     additionalRefs?: (React.MutableRefObject<HTMLDivElement | null> | string)[]
-    style?: React.CSSProperties
-    /** Whether the parent popover should be closed as well on click. Useful for menus  */
```

Comment:
Did you remove this JSDoc comment on purpose?

Reply:
Oops, this got mangled a bit, fixed!